src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -82,14 +82,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
                                                              nullptr));
 
         auto cmdline{ wil::ExpandEnvironmentStringsW<std::wstring>(_commandline.c_str()) }; // mutable copy -- required for CreateProcessW
-
-        til::env environment;
-        auto zeroEnvMap = wil::scope_exit([&]() noexcept {
-            environment.clear();
```

Comment:
Why did we used to do this, anyway?

Reply:
I'm not sure! Maybe security? But that wouldn't have worked, since the creation of the many `std::wstring` instances during `regenerate()` would leave tons of "unsafely" freed memory locations around anyways.